<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="4" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Captureing screenshot]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Jul_23_13_13_39_IST_2023.JPG>Screenshot</a>]]>
    </line>
  </reporter-output>
  <suite started-at="2023-07-23T07:43:39Z" name="Data Driven Test" finished-at="2023-07-23T07:43:45Z" duration-ms="6812">
    <groups>
    </groups>
    <test started-at="2023-07-23T07:43:39Z" name="Login Test" finished-at="2023-07-23T07:43:45Z" duration-ms="6812">
      <class name="com.udemy.testcases.AddCustomerTest">
        <test-method signature="addCustomer(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.udemy.testcases.AddCustomerTest@14a54ef6]" started-at="2023-07-23T07:43:39Z" name="addCustomer" data-provider="getData" finished-at="2023-07-23T07:43:45Z" duration-ms="6379" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Radha]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Raman]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[354356.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Customer added successfully]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomer -->
      </class> <!-- com.udemy.testcases.AddCustomerTest -->
      <class name="com.udemy.testcases.LoginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.udemy.testcases.LoginTest@33a053d]" started-at="2023-07-23T07:43:36Z" name="setUp" finished-at="2023-07-23T07:43:39Z" duration-ms="2537" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginAsBankManager()[pri:0, instance:com.udemy.testcases.LoginTest@33a053d]" started-at="2023-07-23T07:43:39Z" name="loginAsBankManager" finished-at="2023-07-23T07:43:39Z" duration-ms="158" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test Failed
at org.testng.Assert.fail(Assert.java:96)
at com.udemy.testcases.LoginTest.loginAsBankManager(LoginTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Captureing screenshot]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Jul_23_13_13_39_IST_2023.JPG>Screenshot</a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.udemy.testcases.LoginTest@33a053d]" started-at="2023-07-23T07:43:45Z" name="tearDown" finished-at="2023-07-23T07:43:46Z" duration-ms="392" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.udemy.testcases.LoginTest -->
    </test> <!-- Login Test -->
  </suite> <!-- Data Driven Test -->
</testng-results>
